doc/using/InvokingGHDL.rst

```diff
@@ -624,64 +627,64 @@ have been split for readability):
       [std_logic_vector, std_logic_vector return boolean]
   ../translate/ghdldrv/ghdl: compilation error
 
-Indeed, the `"="` operator is defined in both packages, and both are visible at the place it is used.  The first declaration is an implicit one, which occurs when the `std_logic_vector` type is declared and is an element to element comparison, the second one is an explicit declared function, with the semantic of an unsigned comparison.
+Indeed, the `"="` operator is defined in both packages, and both are visible at the place it is used.  The first declaration is an implicit one, which occurs when the `std_logic_vector` type is declared and is an element to element comparison.  The second one is an explicit declared function, with the semantics of an unsigned comparison.
```

Comment:
[Coding Style](https://ghdl.readthedocs.io/en/latest/references/CodingStyle.html#coding-style) says to use "_two spaces at end of sentences_". Otherwise I would've only used one.

There are 270 times in the docs where two spaces are used before a new sentence.

Reply:
Sorry. I missed this, and I think that there are many with a single space. Can we use a regex to force all of them to two spaces?

Reply:
Wow such a rule is coming from GNU ... I saw even some texts using 3 and 4 spaces.

In typography, there are special type setting rules for the end of sentences. Ideally the space behind a sentence delimiter sign is 1.5x as wide as a regular space between words. Since such a space character is not offered by ReST, we should stick to normal spaces. Double spaces are not translated in any markup language to wider spaces. Moreover, double space followed by a linebreak is a forced line break in markdown.

For me it's wrong to use two characters, because the problem of wider spaces must be solved by a typesetting rule. The wider space is then generated by a matching glyph based on the font.

For example, when you translate our documentation to PDF you'll find these wider spaces, because LaTeX is a real typesetting tool obeying to these rules. It's using different glyphs.

Reply:
I would also use single spaces. However, we should rewrite the coding style accordingly.

BTW, it'd be good to put the outcome of the discussion about indentation above in the coding style.

Reply:
Done [here](https://github.com/ghdl/ghdl/pull/591/commits/0490a02c0982962c944e3b8eb7ded090e18e403b)